### PR TITLE
{geo}[intel/2018b] CESM-deps v2 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by vanzod** [comment]
@hajgato I am not familiar with `CESM` but could you explain your approach? Wouldn't it be better to do all the process you describe in the bundle within an easyblock and install a complete version of `CESM`?

**Comment by hajgato** [comment]
needs PR #6916

**Comment by boegel** [comment]
@vanzod See #6675 for more information

File: easybuild/easyconfigs/c/CESM/CESM-2-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description = """CESM is a fully-coupled, community, global climate model that provides state-of-the-art
+ computer simulations of the Earth's past, present, and future climate states."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [
+    ('CMake', '3.11.4'),
+    ('Python', '2.7.15'),
+    ('Perl', '5.28.0'),
+    ('XML-LibXML', '2.0132', '-Pelr-%(perlver)s'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CESM/CESM-2-intel-2018b.eb]: @hajgato `Pelr` -> `Perl`

**Comment by hajgato** [comment]
@boegel Renamed to `CESM-deps`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3146.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/73c0f15754162b856335a625e1803a6e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in this PR)
node2095.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0d6580dd0b46e6ab1a5b65ad285da77d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in this PR)
node2586.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/17d0d816d8be5d14d18ac66b6c2a93bd for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**